packages/server-common/src/handlers/user/sync-active-campaign-contact-factory.ts

```diff
@@ -24,8 +36,144 @@ export interface UserController {
   ): Promise<gp2.UserResponse | UserResponse>;
 }
 
+export const syncUserActiveCampaignData = async (
```

Comment:
this function's signature is insane, make sure to use an object with config for some of the parameters like tokens, etc, you should also consider not passing all functions as dependencies (especially if they are pure with no side effects